src/plugin.test.ts

```diff
@@ -0,0 +1,5 @@
+import plugin from './plugin'
+
+describe('main plugin', () => {
+  it.todo('write a test')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/richardvanbergen/payload-plugin-cloud-storage/pullRequest?prid=7829468)
```suggestion
  it.todo("write a test")
```